app/presenters/summary/sections/other_capital_details.rb

```diff
@@ -10,7 +10,7 @@ def answers
           Components::ValueAnswer.new(
             :has_frozen_income_or_assets, capital.has_frozen_income_or_assets,
             change_path: edit_steps_capital_frozen_income_savings_assets_capital_path
-          ),
+          )
```

Comment:
Are we not displaying the value of `has_no_other_assets` in the review application screen?